src/Serialization/main.jl

```diff
@@ -166,7 +166,7 @@ end
 
 ################################################################################
 # Default generic save_internal, load_internal
-function save_internal(s::SerializerState, obj::T) where T
+function save_internal_generic(s::SerializerState, obj::T) where T
     result = Dict{Symbol, Any}()
     for n in fieldnames(T)
         if n != :__attrs
```

Comment:
Why don't you want to serialise the attributes?

Reply:
Good question. But I deliberately didn't want to touch it with this PR, as it is orthogonal to the concern this PR is supposed to address :-)

Reply:
I skipped them as they didn't work (for toric varieties). I'd like to have them in the future.